polkadot-protocol/introduction/glossary.md

```diff
@@ -70,12 +70,15 @@ They aren't [authorities](#authority) in themselves, as they require a [relay ch
 
 More details are found on the [Polkadot Collator Wiki](https://wiki.polkadot.network/docs/learn-collator){target=\_blank}.
 
-## Consensus
+## Collective
+
+Most often used to refer to an instance of the [Collective pallet](#collective) on Polkadot SDK-based networks such as [Kusama](#kusama) or [Polkadot](#polkadot) if the Collective pallet is part of the [Framework for Runtime Aggregation of Modularized Entities](#frame-framework-for-runtime-aggregation-of-modularized-entities) (FRAME)-based [runtime](#runtime) for the network.
```

Comment:
The `Collective pallet` link seems to be wrong because it points to Collective itself. Also, the first sentence feels too long.